.github/workflows/release.yaml

```diff
@@ -0,0 +1,21 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name : Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      - name: Create release on Github
+        run: |
+          gh --repo "${{ github.repository }}" release create ${{ github.ref_name }} --verify-tag --generate-notes
```

Comment:
Of course for the "example" case I picked the simplest that didn't require dealing with RCs.. Woops.

It would be great to detect whether the tag pushed is an RC and if so create a pre release instead of a release. This is done using the `--prerelease` flag like `gh --repo <repo> release --prerelease`. The tag pushed is `${{ github.ref_name }}` so you could detect `-rc` in it and add the flag if it is.

Reply:
Is the check I added sufficient?  Or do we need to be more specific?

Reply:
That seems to work :+1:  Thanks!